lib/routemap.h

```diff
@@ -142,6 +142,9 @@ struct route_map_index {
 	struct route_map_index *next;
 	struct route_map_index *prev;
 
+	/* Keep track how many times we've try to apply */
+	uint64_t applied;
```

Comment:
Just a nitpick. "applied" sounds like a boolean to me which says whether the route map is applied or not. Where as we are actually storing the count or number of times this route map was applied.

Reply:
Argh my comment got posted in a separate review entry...

It's called "sequence" in the CLI help text too:

```
DEFUN_NOSH (route_map,
       route_map_cmd,
       "route-map WORD <deny|permit> (1-65535)",
...
       "Sequence to insert to/delete from existing route-map entry\n")
```

I think sequence makes sense too since we're applying route-map entries "in order of the sequence".

Plus if we change it, please change it everywhere :)

Reply:
ok I'll leave it as sequence.

Reply:
as for applied as a boolean, won't it be obvious once someone actually looks at the code?  I was having a bit of a a problem figuring out what to call it.